dowhy/causal_estimator.py

```diff
@@ -14,12 +14,15 @@ class CausalEstimator:
     """
     # The default number of simulations for statistical testing
     DEFAULT_NUMBER_OF_SIMULATIONS_STAT_TEST = 1000
-    # The default number of simulations to obtain confidence intervals
+    # The default number of simulations to obtain confidence interval
     DEFAULT_NUMBER_OF_SIMULATIONS_CI = 100
     # The portion of the total size that should be taken each time to find the confidence interval
     # 1 is the recommended value 
     # https://ocw.mit.edu/courses/mathematics/18-05-introduction-to-probability-and-statistics-spring-2014/readings/MIT18_05S14_Reading24.pdf
-    DEFAULT_SAMPLE_SIZE = 1
+    # https://projecteuclid.org/download/pdf_1/euclid.ss/1032280214 
+    DEFAULT_SAMPLE_SIZE_FRACTION = 1
+    # The default Confidence Level
+    DEFAULT_CONFIDENCE_LEVEL = 0.05
```

Comment:
Oh, I may have miswritten. The confidence level is defined as 0.95, not 0.05.

Reply:
Makes sense. I'll add it to the commit.